simulation_parameters/SimulationParameters.cs

```diff
@@ -106,6 +106,11 @@ public IEnumerable<OrganelleDefinition> GetAllOrganelles()
         return organelles.Values;
     }
 
+    public bool HasOrganelle(string name)
```

Comment:
I think this should be named something like "DoesOrganelleExist" as I think HasOrganelle is too similar to the actual placed organelles in an organelle layout, so fundamentally I think the SimulationParameters containing an organelle is a different thing.